models/User.go

```diff
@@ -1041,6 +1077,65 @@ func PerformBuyFromExchangeTransaction(userId uint32, stockId uint32, stockQuant
 		return nil, fmt.Errorf(format, args...)
 	}
 
+	// Tax Calculation
```

Comment:
This is done. I've created 2 helper functions - one for bidding user, and one for asking user. I decided not to combine the 2 as that would've complicated things more rather than simplified it.